lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -1188,6 +1190,7 @@ public void testRefreshing()
         accessControl.checkCanCreateView(alice, aliceView);
         accessControl.checkCanCreateView(alice, aliceView);
 
+        //noinspection UnstableApiUsage
```

Comment:
We should suggest to disable this inspection. A lot of guava is using `@Beta` that causing this inspection to trigger. Let's update https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md instead.

Reply:
#9395